share/completions/btrbk.fish

```diff
@@ -0,0 +1,62 @@
+# Filter Completion
+function __fish_btrbk_complete_filter
+    btrbk list config --format col:h:snapshot_name,source_subvolume,target_url | string replace -r '([^ ]+)\s+([^ ]+)\s+([^ ]*)' '$1\t$2 -> $3'
+end
+
+# options with arguments
+complete -c btrbk -l format        -x -d 'Change output format'                     -a "table long raw"
```

Comment:
AFAIS, `--format` is allowed just before any subcommand. I would rather add `-n` flag to `complete` to restrict when `--format` is shown, as well as other options falling in this category.

My comment is based on [this](https://manned.org/man/btrbk) man page:

```
       btrbk [-h|--help] [--version]
             [-c|--config <file>] [-n|--dry-run] [--exclude <filter>]
             [-p|--preserve] [--preserve-snapshots] [--preserve-backups]
             [-v|--verbose] [-q|--quiet] [-l|--loglevel <level>]
             [-t|--table] [-L|--long] [-1|--single-column]
             [--format <output-format>] [--pretty]
             [-S|--print-schedule] [--progress]
             [--lockfile <file>]
             [--override <config_option>=<value>]
             <command> [[--] <filter>...]
```